coordinator/src/services/verification/signature-verification.ts

```diff
@@ -5,7 +5,7 @@ import {convertFragmentsToHashableFormat, extractFragments} from "../translation
 import {createParametersDigest} from "../translation/request"
 import crypto from "crypto"
 import {isTruthy} from "../translation/common"
-
+import {convertHL7V3DateTimeToIsoDateTimeString, isDateInRange} from "../translation/common/dateTime"
 function verifySignature(parentPrescription: hl7V3.ParentPrescription): Array<string> {
```

Comment:
As mentioned earlier, we could rename this to `verifyPrescriptionSignature`

Reply:
thanks good suggestion method renamed